external/docs/docs.depproj

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier>none</NuGetRuntimeIdentifier>
+    <OutputPath>$(XmlDocDir)</OutputPath>
+    <IsRuntimeAssembly>false</IsRuntimeAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="$(XmlDocPackage)">
+      <Version>$(XmlDocPackageVersion)</Version>
+    </PackageReference>
+  </ItemGroup>
+
+  <!-- the intellisense package doesn't use nuget conventions so we need to select manually -->
+  <Target Name="GetDocFiles" AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <DocFile Include="$(XmlDocFileRoot)/**/*.xml"/>
+      <DocFile>
+        <!-- trim off slash since it differs by platform and we need to do a string compare -->
+        <LCID>$([System.String]::new('%(RecursiveDir)').TrimEnd('\').TrimEnd('/'))</LCID>
```

Comment:
Actually it doesn't repro when using the latest core msbuild.

I created the test project:
```
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup>
    <_testString>abc\</_testString>
  </PropertyGroup>
  <Target Name="Build">
    <Message Importance="High" Text="&quot;$(_testString)&quot;.IndexOfAny(&quot;\/&quot;) = $(_testString.IndexOfAny('\/'))" />
    <Message Importance="High" Text="&quot;$(_testString)&quot;.TrimEnd(&quot;\/&quot;) = $(_testString.TrimEnd('\/'))" />
  </Target>
</Project>
```

This fails when run on our tool-runtime msbuild:
```
MSBuild.sh /nologo /v:m
  "abc\".IndexOfAny("\/") = 3
/Users/ericstj/src/test/test.proj(8,32): error MSB4184: The expression ""abc\".TrimEnd(\/)" cannot be evaluated. String must be exactly one character long.
```
But succeeds when run on latest:
```
dotnet msbuild /nologo /v:m
  "abc\".IndexOfAny("\/") = 3
  "abc\".TrimEnd("\/") = abc
```

I guess we can clean this up when we update to a newer core-msbuild.